sample-plugin/pom.xml

```diff
@@ -1177,22 +1192,49 @@
       <id>2.462.x</id>
       <properties>
         <bom>2.462.x</bom>
+        <hpi-plugin.version>3.57</hpi-plugin.version>
         <jenkins.version>2.462.3</jenkins.version>
+        <jenkins-test-harness.version>2225.2230.v6210cb_b_827f9</jenkins-test-harness.version>
+        <maven.compiler.release>11</maven.compiler.release>
       </properties>
     </profile>
     <profile>
       <id>2.452.x</id>
       <properties>
         <bom>2.452.x</bom>
+        <hpi-plugin.version>3.57</hpi-plugin.version>
         <jenkins.version>2.452.3</jenkins.version>
+        <jenkins-test-harness.version>2225.2230.v6210cb_b_827f9</jenkins-test-harness.version>
+        <maven.compiler.release>11</maven.compiler.release>
       </properties>
+      <dependencyManagement>
+        <dependencies>
+          <dependency>
+            <groupId>jakarta.servlet</groupId>
+            <artifactId>jakarta.servlet-api</artifactId>
+            <version>4.0.4</version>
+          </dependency>
+        </dependencies>
+      </dependencyManagement>
     </profile>
     <profile>
       <id>2.440.x</id>
       <properties>
         <bom>2.440.x</bom>
+        <hpi-plugin.version>3.57</hpi-plugin.version>
         <jenkins.version>2.440.3</jenkins.version>
+        <jenkins-test-harness.version>2225.2230.v6210cb_b_827f9</jenkins-test-harness.version>
+        <maven.compiler.release>11</maven.compiler.release>
       </properties>
+      <dependencyManagement>
+        <dependencies>
+          <dependency>
+            <groupId>jakarta.servlet</groupId>
+            <artifactId>jakarta.servlet-api</artifactId>
+            <version>4.0.4</version>
```

Comment:
I suppose #3685 was unwanted?

Reply:
Yes, I think it is unwanted.  Attempt to revert it and prevent new arrivals in:

* #3686